1.0/docs/devguide/styling.md

```diff
@@ -699,3 +699,21 @@ that includes a `custom-style` element.
       } 
     </style>      
 
+## Third-party libraries that modify local DOM
```

Comment:
Please add a short ID for this header, like `{#scope-subree}`. I expect people will be linking to it, and we don't want links to break if the header text changes a little later on.

(Background info: at some point, planning to add an containing section on working with third-party libraries & frameworks, so we can cover other cases, like patch-dom. It's possible that might necessitate tweaking this header text.)